frontend-web/webclient/app/Applications/Jobs/View.tsx

```diff
@@ -402,13 +402,13 @@ const Content = styled.div`
 `;
 
 const InQueueText: React.FunctionComponent<{job: Job}> = ({job}) => {
-    const [utilization, setUtilization] = useCloudAPI<compute.RetrieveUtilizationResponse | null>(
+    const [utilization, setUtilization] = useCloudAPI<compute.JobsUtilizationResponse | null>(
         {noop: true},
         null 
     );
 
     useEffect(() => {
-        setUtilization(compute.jobs.retrieveUtilization())
+        setUtilization(compute.jobs.utilization())
```

Comment:
Is this generated automatically? If this fetches from a backend and isn't just a getter, I think the function-name obscures some information.

Reply:
I seems so. Consider the frontend part accepted from me.

Reply:
I see I forgot to change the name of the end-point, which I assume is why it generated that function name. It should be fixed now, back to `retrieveUtilization`.